esp-wifi/src/lib.rs

```diff
@@ -109,7 +109,7 @@ const DEFAULT_TICK_RATE_HZ: u32 = 100;
 struct Config {
     #[default(5)]
     rx_queue_size: usize,
-    #[default(3)]
+    #[default(10)]
```

Comment:
I'm not sure if we should change the default just to get rid of some warnings

We probably will revise the defaults at some point (e.g. check https://github.com/esp-rs/esp-wifi/pull/233#pullrequestreview-1569469944 ) and when we do it, we also need to update the documentation about it

Especially since this is user-configurable

Reply:
I think we can keep default value as #233. We can change when using because it user-configurable